packages/react-ai/src/ai-conversation-styles.css

```diff
@@ -166,6 +166,7 @@
   align-self: stretch;
   overflow: hidden;
   resize: none;
+  outline: none;
```

Comment:
Just leaving a note here for posterity: we probably shouldn't remove the outline for a11y on these default elements?

Reply:
makes sense, fixed